docs/rollout/introduce-tilt.md

```diff
@@ -1,63 +1,80 @@
 
-## You'll Like Tilt More Because...
+Here's our recommended format (we'll dig into each section below):
 
-Start with why Tilt is better, and focus on why it's better for the AppDev right now. A piece of perspective: some Tilt benefits are more exciting to the DevEx who have to support the tooling than to the AppDevs using it. Instead of leading with attributes like "easier to maintain" or "closer to prod", we'd suggest focusing on advantages like being faster, having a better UI, or supporting new workflows.
+```
+1. You'll Like Tilt More Because...
+2. How to Get Started
+  2a. How to Install Tilt
+  2b. What to Do After You Start Tilt
+3. Known Issues
+4. What to Do Next
+```
 
-Highlight benefits of Tilt for your project compared to the legacy tool. Don't just list features of Tilt: some may not be new compared to the legacy tool and some you may not use yet.
+Remember that a Transition Guide is more than just a README. Its job is to get an AppDev up and running with minimal friction, but also to drive home why they should be excited about the new tool.
 
-Include a screenshot to make it easy to understand the difference. (If your team uses Tilt Cloud, you can also use [Snapshots](../snapshots) to let potential users interact with the UI before installing Tilt).
+It's also not a design doc or implementation proposal: focus on what makes this tool compelling to a day-to-day developer, not to a DevEx team (e.g. "easy to maintain") or a VP of Eng (e.g. "will boost overall productivity by X%").
 
-## How To Get Started
+### You'll Like Tilt More Because...
 
-Describe how to install Tilt and any prereqs you need.
+Start with why Tilt is better than the current way of doing things. Don't just list features of Tilt: highlight Tilt's benefits as compared to the legacy tool.
 
-### What To Do After You Start Tilt
-It's useful after you say how to start Tilt to give a simple example workflow that shows off getting quick feedback with Tilt. For example:
+This part of the doc can also be a great place to re-contextualize or justify what AppDevs might see as sticking points. For instance, if initial `tilt up` is noticeably slower than launching your old tool, consider a note like:
+> You may notice that Tilt is a little slower to start up than $OLDTOOL, but if you stick with it, you'll see that once your app is up and running, _updates_ are 10x faster than on $OLDTOOL.
+
+A Transition Guide is about making your case to an AppDev, so focus on the features that most benefit someone who writes code day-to-day. Instead of leading with attributes like "easier to support users" or "dev is closer to prod." (which are more exciting to those maintaining a tool than those using it), we'd suggest focusing on advantages like being faster, having a better UI, or supporting new workflows.
```

Comment:
Maybe something like "if you have focus users (test pilots?) (with a link), consider asking them how Tilt is particularly useful for your team"

Reply:
ah, I see you mentioned something like this down under "advanced features"...I feel like it's better here, or maybe fine in both, but then should probably adjust the wording of the second instance